### ISSUE TITLE
fatal error on new static resource creation

### ISSUE THREAD

**Issue by feamsr00** [modxbot/migrate#3167]
feamsr00 created Redmine issue ID 3167

Hello There,

Any time I create a static resource, php produces the following error:

<pre>
Fatal error: Cannot pass parameter 1 by reference in [...]/manager/controllers/default/resource/staticresource/create.php on line 150
</pre>

This hasn't proved to be a major issue as I can still "Quick Create" static resources. 

Please let me know if there is any other information you need to troubleshoot this issue. TIA

**Comment by feamsr00** [comment]
feamsr00 submitted:

Also, this _might_ be related to issue #3074, but I haven't had too many other issues after upgrading 2.0.4->2.0.6

**Comment by splittingred** [comment]
splittingred submitted:

Sounds like you didn't upgrade fully; files were not copied. Did you make sure to uncheck the two boxes on the options page during setup?

**Comment by feamsr00** [comment]
feamsr00 submitted:

Actually, it was an "advanced" install. The installer _created_ the manager folder. I only kept a couple "config key" files so I could get a db upgrade. Also, I'm wondering if the underlying is issue is causing #3170.

Update: I took someone's suggestion of recopying the files from an "Traditional" package and this seemed to not only correct the issue, but changed the manager theme/icons a bit. I would ask if there was a distro issue, but there's a already new version out.
